src/components/Card/index.tsx

```diff
@@ -203,11 +206,23 @@ const TopLeftAdornment = styled.div`
   }
 `
 
+const Period = styled.div`
+  display: inline-flex;
+  align-items: center;
+`
+
+const PeriodText = styled(TinyText)`
+  padding-inline-start: ${space[4]};
+
+  :first-letter {
+    text-transform: capitalize;
+  }
+`
+
```

Comment:
You shouldn't use padding to create space between elements, but margin 😬 

Btw if you use `flex`, you could use `gap` instead 😄